src/database/migrations/20210106143948-create-positions-table.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("positions", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.TINYINT,
```

Comment:
- I would put here INTEGER, we don't want to restrict to just 255 positions in the DB. TYNYINT is generally used for boolean to store just 0 and 1 (https://sequelize.org/master/class/lib/data-types.js~BOOLEAN.html). Also, make it UNSIGNED.